vpc/service/branch_enis_test.go

```diff
@@ -63,7 +63,7 @@ func TestListBranchToTrunkENIMapping(t *testing.T) {
 	mock.ExpectQuery("SELECT branch_eni, trunk_eni FROM branch_eni_attachments WHERE state = 'attached'").WillReturnRows(rows)
 	mock.ExpectCommit()
 
-	service := vpcService{db: db}
+	service := VpcService{db: db}
```

Comment:
Does `VpcService` have to be exported? (capitalized)
When you export stuff from other packages, you lose the ability for go linters to let you know if you have dead code in certain situations

Reply:
I changed it so that any users of `VpcService` needs to construct the struct first and then use the `Run` method to start it. I think it needs to be exported for `main.go` (in `cmd` package) to use it. Do you know if there is a way in Golang to keep it private but also visible to `cmd/titus-vpc-service`?

Reply:
The only way I know how to do it is to make the cmd main.go be as minimal as possible, and then put everything in vpc/service folder

Reply:
Alright. Changed back to `vpcService` for better lints. Well, I have to say, coming from C++ and Rust world, Go's visibility controlled by naming thing bothers me a little.